service.go

```diff
@@ -50,12 +57,19 @@ type (
 		// to "/assets/x/y/z".
 		ServeFiles(path, filename string) error
 
-		// ServeMux returns the service request mux.
-		ServeMux() ServeMux
-
 		// NewController returns a controller for the resource with the given name.
 		// This method is mainly intended for use by generated code.
 		NewController(resName string) Controller
+	}
+
+	// Version is the interface for interacting with individual versions. It is embedded by
+	// application for default use with versionless apps
+	Version interface {
+		// Version returns the version string ID
+		Version() string
```

Comment:
This is a lot of versions, but GetVersion("1.0").Name() seemed weird. Even in comments, I'm not quite sure how to refer to the stringified version. Is is a `VersionID`? `VersionName`? Just `Version`?